src/adsp_config.c

```diff
@@ -1915,6 +2268,72 @@ static int parse_adsp_config_v2_5(const toml_table_t *toml, struct adsp *out,
 	return 0;
 }
 
+static int parse_adsp_config_v2_6(const toml_table_t *toml, struct adsp *out,
```

Comment:
Whats the difference here with v2.5 ? My understanding is that only the ext manifest is different ?